Libraries/Blob/URL.js

```diff
@@ -43,13 +42,143 @@ if (BlobModule && typeof BlobModule.BLOB_URI_SCHEME === 'string') {
  *
  * ```xml
  * <resources>
- *   <string name="blob_provider_authority">your.app.package.blobs</string>
+ *   <string name='blob_provider_authority'>your.app.package.blobs</string>
  * </resources>
  * ```
  */
-class URL {
-  constructor() {
-    throw new Error('Creating URL objects is not supported yet.');
+
+// Small subset from whatwg-url: https://github.com/jsdom/whatwg-url/tree/master/lib
+// The reference code bloat comes from Unicode issues with URLs, so those won't work here.
+export class URLSearchParams {
```

Comment:
The class is missing the methods `delete(name)`, `get(name)`, `getAll(name)`, `has(name)` and `set(name, value)`. Maybe we should the similar to `URL` and add the methods which throw a proper error when invoked?